Tests/Quality/error_qual_space.fastq

```diff
@@ -18,4 +18,3 @@ YYYYYYYYYYYYYYYYYY WWWYWY
 GTATTATTTAATGGCATACACTCAA
 +SLXA-B3_649_FC8437_R1_1_1_183_714
 YYYYYYYYYYWYYYYWYWWUWWWQQ
-
```

Comment:
True the blank line isn't needed, and in hindsight ought not to be there, but this is the file as included in the paper's supplementary materials so I would prefer not to change it: 
https://academic.oup.com/nar/article/38/6/1767/3112533